lib/engines/hcktest/tests.rb

```diff
@@ -249,10 +249,22 @@ def summary_results_log
       @tests.reduce('') do |sum, test|
         extra_info = @tests_extra.dig(test['id'], 'dump') ? '(with Minidump)' : ''
         status = @tests_extra.dig(test['id'], 'status') || test['status']
-        sum + "#{status}: #{test['name']} [#{test['estimatedruntime']}] #{extra_info}\n"
+        queued_at, started_at = format_times(test)
+
+        sum + "#{status}: #{test['name']} [#{test['estimatedruntime']}] #{extra_info}#{queued_at}#{started_at}\n"
       end
     end
 
+    def format_times(test)
+      queued_at = @tests_extra.dig(test['id'], 'queued_at')&.strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
After `strftime` you don't have a timezone in the string, so it is not clear what exactly the time is when CI has a different timezone from you. Don't format time as we don't do this in any user-visible reports.